core/cppgc.rs

```diff
@@ -212,4 +214,19 @@ impl<T: GarbageCollected + 'static> SameObject<T> {
       })
       .clone()
   }
+
+  pub fn set(
```

Comment:
when is this useful? sameobject should always just be set once and forget, any reason `get` cannot be used in your scenario?

Reply:
Although `get` is sufficient, I think it may be useful when we want to assign a value in advance in a constructor or static method.
https://github.com/denoland/deno/blob/dfa4af12b093ac3a504d7fe98004ad7affd1c91a/ext/geometry/lib.rs#L311-L315

Reply:
makes sense